src/main/java/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityRootAction.java

```diff
@@ -661,4 +603,24 @@ public static QueryMode fromString(@CheckForNull String str) {
             }
         }
     }
+    
+    /**
+     * Gets the {@link DockerTraceabilityRootAction} of Jenkins instance.
+     * @return Instance or null if it is not available
+     */
+    public static @CheckForNull DockerTraceabilityRootAction getInstance() {
+        Jenkins j = Jenkins.getInstance();
+        if (j == null) {
+            return null;
+        }
+        
+        @CheckForNull DockerTraceabilityRootAction action = null;
+        for (Action rootAction : j.getActions()) {
+            if (rootAction instanceof DockerTraceabilityRootAction) {
+                action = (DockerTraceabilityRootAction) rootAction;
```

Comment:
Easier to use `j.getExtensionList(RootAction.class).get(DockerTraceabilityRootAction.class)`. Or, assuming a sufficiently new core version, just use `ExtensionList.lookup` and omit any explicit call to `Jenkins.getInstance`.